templates/user/dashboard/issues.tmpl

```diff
@@ -89,7 +89,7 @@
 						{{if .Repo}}
 						<li class="item">
 							<div class="ui label">{{.Repo.FullName}}#{{.Index}}</div>
-							<a class="title has-emoji" href="{{AppSubUrl}}/{{.Repo.Owner.Name}}/{{.Repo.Name}}/issues/{{.Index}}">{{.Title}}</a>
+							<a class="title has-emoji" href="{{.HTMLURL}}">{{.Title}}</a>
```

Comment:
What it means?

Reply:
In the old days the link (URL) for an issue was calculated in the templates from its constitutive parts. `.HTMLURL` maps to `(Issue*) HTMLURL() string` which already returns the right value (uses `/pulls/` instead of `/issues/` if it's a PR). There are still some templates like this that have the old ways, so we need to update them.